core/src/main/java/org/mifos/mobilewallet/core/domain/model/Transaction.java

```diff
@@ -161,4 +161,142 @@ public String toString() {
                 ", receiptId=" + receiptId +
                 '}';
     }
+
+    private static String input;
+    private static String[] units=
+            {"",
+                    " One",
+                    " Two",
+                    " Three",
+                    " Four",
+                    " Five",
+                    " Six",
+                    " Seven",
+                    " Eight",
+                    " Nine"
+            };
+    private static String[] teen=
+            {" Ten",
+                    " Eleven",
+                    " Twelve",
+                    " Thirteen",
+                    " Fourteen",
+                    " Fifteen",
+                    " Sixteen",
+                    " Seventeen",
+                    " Eighteen",
+                    " Nineteen"
+            };
+    private static String[] tens=
+            { " Twenty",
+                    " Thirty",
+                    " Forty",
+                    " Fifty",
+                    " Sixty",
+                    " Seventy",
+                    " Eighty",
+                    " Ninety"
+            };
+    private static String[] maxs=
+            {"",
+                    "",
+                    " Hundred",
+                    " Thousand",
+                    " Lakh",
+                    " Crore"
+            };
+
+
+    public String convertNumberToWords(int n) {
```

Comment:
@shobhit24 Instead of implementing the method inside the `core` module, you can create a utility class with the same functionality inside the `app` module. Also please fix the checkstyle errors

Reply:
Okay sir, I'll implement it in the utility class.